src/LibraryPage.tsx

```diff
@@ -39,23 +40,41 @@ const LibraryPage: React.FC = () => {
   const [modalOpen, setModalOpen] = useState(false);
   const [isNewBook, setIsNewBook] = useState(true);
   const [searchTerm, setSearchTerm] = useState("");
+  const [catSelected, setCatSelected] = useState<{
+    label: string;
+    value: string;
+  }>();
```

Comment:
Should this be a separate type in ourtypes? It's also defined in the BookForm file.

Reply:
Yeah :P that's probably a good idea. I'll put it in there.